internal/feedback/feedback.go

```diff
@@ -73,7 +73,7 @@ func (cf ConstraintFeedback) LogFeedback(logger *log.Logger) {
 	if cf.Constraint != "" {
 		logger.Printf("  %v", GetUsingFeedback(cf.Constraint, cf.ConstraintType, cf.DependencyType, cf.ProjectPath))
 	}
-	if cf.LockedVersion != "" && cf.Revision != "" {
+	if cf.Revision != "" {
```

Comment:
Hmm, actually this shouldn't have any impact on constraint.  Can you clarify what kind of test I should add there?